aspnetcore/release-notes/aspnetcore-10/includes/OpenApiPopulateXMLDocComments.md

```diff
@@ -10,6 +10,8 @@ ASP.NET Core OpenAPI document generation will now include metadata from XML doc
 
 At build-time, the OpenAPI package will leverage a source generator to discover XML comments in the current application assembly and any project references and emit source code to insert them into the document via an OpenAPI document transformer.
 
+Support for generating OpenAPI metadata from XML documentation comments has been extended with ASP.NET Core in .NET 10 to extract metadata for operation responses from `<returns>` and `<response>` XML tags.
+
```

Comment:
I don't think we need this.  All support for extracting OpenAPI metadata from XML docs was added in .NET 10, with most of that done in earlier previews. Since the prior paragraph is vague about what specific XML tags are supported, I don't think we need to call out the added support for these two tags that was added in preview5.
```suggestion
```